src/index.ts

```diff
@@ -27,7 +27,8 @@ function loader(this: interfaces.Webpack, contents: string) {
     }
 
     const rawFilePath = path.normalize(this.resourcePath);
-    const filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
+    let filePath = utils.appendTsSuffixIfMatch(options.appendTsSuffixTo, rawFilePath);
```

Comment:
Right - thanks for that.  Is it likely that people will use `appendTsxSuffixTo` **and** `appendTsSuffixTo` together?  Or is it an either / or thing?